framework/core/js/src/common/utils/getContrast.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * The `getContrast` utility converts a hex color to rgb, and then calcul a YIQ
+ * value in order to get the appropriate contrast value (is it dark or is it
+ * light?) See https://www.w3.org/TR/AERT/#color-contrast for references
+ */
+
+export default function getContrast(hexcolor: String) {
```

Comment:
I think we should export a second `isDark` util here to avoid repeating `>= 128` and avoid using this magic number through our code.